src/Glpi/Api/HL/Controller/AssetController.php

```diff
@@ -900,6 +961,45 @@ public static function getAssetTypes(bool $classes_only = true): array
         return $classes_only ? array_keys($assets) : $assets;
     }
 
+    public static function getRackTypes(bool $schema_names_only = true): array
```

Comment:
Why not relying on `$CFG_GLPI['rackable_types']`?

Reply:
The main reason is stability. The array could be changed later in GLPI itself or by plugins and the names of API schemas don't necessarily match the class names (Would be a really bad experience with namespaced and plugin classes).

If plugins add new rackable types, they should also register a new API controller with routes that can cover those new types.